src/main/resources/META-INF/plugin.xml

```diff
@@ -2,6 +2,12 @@
     <id>com.redhat.devtools.intellij.kubernetes</id>
     <name>Kubernetes by Red Hat</name>
     <vendor email="developers@redhat.com" url="http://www.redhat.com">Red-Hat</vendor>
+    <description>
+        <![CDATA[
+        <p>A plugin for interacting with Kubernetes and OpenShift clusters.</p>
+        <p>The plugin provides functionalities and user experiences that are very close to the Kubernetes extension for vscode (Visual Studio Code Kubernetes Tools).</p>
```

Comment:
You did not get me: I was about the reference to VSCode (concurrent IDE) and not the overview title !!!
Please restore it

Reply:
Add lt's see if the reference to VSCode pass the approval

Reply:
I'd remove **Overview** and the icon as both repeat what the page already features (see screenshots & explanations above). I also think that referring to the vscode IDE should be included if possible. As such I'd submit as it currently is. Agree?